rules/framework.bzl

```diff
@@ -528,6 +538,7 @@ def _copy_swiftmodule(ctx, framework_files):
         swiftdoc = outputs.swiftdoc,
         swiftmodule = outputs.swiftmodule,
         swiftinterface = outputs.swiftinterface,
+        symbol_graph = outputs.symbol_graph,
```

Comment:
Changes LGTM 👍 .

Seems like plumbing in the `.symbolgraph` here good enough to get it correctly propagating end to end for you?

Reply:
Yep this worked in testing, it correctly forwards the libraries symbol graphs (if they were requested)